### ISSUE TITLE
Make snapshot builds available

### ISSUE THREAD

**Issue by pluradj** [JanusGraph/janusgraph#40]
https://twitter.com/funkatron/status/819972558871293952

Making snapshot distribution zips available before the first official release will lower the bar for people wanting to try out JanusGraph.

**Comment by funkatron** [comment]
Thanks! That will help a lot, esp for folks who are not Java-savvy.

**Comment by rafatuita** [comment]
can you guys prioritize this issue @pluradj ? quite a few of us can't migrate from titan without snapshots available. Since this is a fork from latest version, don't see the issue to release it as is...

looking forward to move our code to Janus...

**Comment by twilmes** [comment]
Ok, made some progress but still had to deploy the packaged zip manually via `mvn deploy:deploy-file` command line.  I'm continuing to work that so a `mvn deploy` will suffice.

In the meantime though:
* https://oss.sonatype.org/content/repositories/snapshots/org/janusgraph/
* installation zip can be found [here](https://oss.sonatype.org/content/repositories/snapshots/org/janusgraph/janusgraph-dist/0.1.0-SNAPSHOT/)

**Comment by mbrukman** [comment]
@twilmes — thanks for building and deploying the snapshot!

Can you please create a tag at the same Git hash that you built the snapshot from? E.g., `v0.1.0-SNAPSHOT-2017-xx-yy` (with `xx-yy` being the date of the hash)? That would make it possible to create a source release mark on GitHub, and we can attach the binary artifacts on GitHub to host it there for ease of downloading.

Our docs currently send folks to the GitHub release page, so this will make it very easy for users to discover the downloadable artifacts, and get either the source at the same version or the built JARs.

**Comment by twilmes** [comment]
Good call @mbrukman.  I'll take care of that.  That naming scheme sounds good to me.

**Comment by twilmes** [comment]
I got things setup and uploaded the zip to https://github.com/JanusGraph/janusgraph/releases/tag/0.1.0-SNAPSHOT-2017-03-18.  The verbiage is pretty minimal at this point but can be easily updated if we so desire.

**Comment by mbrukman** [comment]
Per discussion on the mailing list, I created a tag [`v0.1.0-rc1`](https://github.com/JanusGraph/janusgraph/releases/tag/v0.1.0-rc1) and removed the earlier tag `0.1.0-SNAPSHOT-2017-03-18` from the repo and releases.

I've also created a release and re-uploaded the `janusgraph-dist-[...].zip` file (renamed to `janusgraph-dist-0.1.0-rc1.zip`) from the previous release so it's a complete replacement of the original.

Closing this issue as we have addressed the original request.
